### PR TITLE
ovl_Obj_Milk_Bin OK

### PR DESCRIPTION
**Pull request by AngheloAlf**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

I made an enum for the params of this actor, but I'm not completely sure how to name them, so I gaved them generic names. Suggestions for those are welcome.

Btw, for some reason, `format.sh` modified files I didn't touch. dunno what happened there.

### CODE REVIEW

**Review by jpburnett — APPROVED** [review]
Once the warnings are fixed, I think this will look legen-dairy

File: src/overlays/actors/ovl_Obj_Milk_Bin/z_obj_milk_bin.c [hunk 1]
```diff
@@ -21,12 +26,80 @@ const ActorInit Obj_Milk_Bin_InitVars = {
...
+
+    if (!(this->disableDraw & 1)) {
+        CollisionCheck_SetAC(globalCtx, &globalCtx->colCheckCtx, &this->collider.base);
+        CollisionCheck_SetOC(globalCtx, &globalCtx->colCheckCtx, &this->collider.base);
+    }
+}
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_Milk_Bin_0x80C08E40/ObjMilkBin_Destroy.asm")
+// gMilkBinMilkJarDL
+extern Gfx D_060004B0[];
```
Comment (zbanks) [inline src/overlays/actors/ovl_Obj_Milk_Bin/z_obj_milk_bin.c:97]: Really small nit: I would put any declarations like this at the top of the file, before any of the definitions?

**Review by zbanks — CHANGES_REQUESTED** [review]
Looks great! :cow2: 

Only one small suggestion (placement of `extern Gfx ...`)

Note: It looks like `z_collision_check.c` was reformatted because it wasn't reformatted after the `s/ctxt/globalCtx/g` change (or was reformatted with a different version of `clang-format`) - so I think it makes sense to reformat it here. (Although no one else needs to reformat it in their PRs, to avoid merge conflicts)

File: src/overlays/actors/ovl_Obj_Milk_Bin/z_obj_milk_bin.c (hunk 1 above)
Comment (AngheloAlf) [reply src/overlays/actors/ovl_Obj_Milk_Bin/z_obj_milk_bin.c]: Done :D

**Review by zbanks — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**